contrib/coredns/policy/policy.go

```diff
@@ -149,7 +178,29 @@ func (p *PolicyMiddleware) ServeDNS(ctx context.Context, w dns.ResponseWriter, r
 	}
 
 	if response.Permit {
-		return middleware.NextOrFailure(p.Name(), p.Next, ctx, w, r)
+		status,err := middleware.NextOrFailure(p.Name(), p.Next, ctx, w, r)
+		if err != nil {
+			return status, err
+		}
+
+		responseattrs := []*pb.Attribute{&pb.Attribute{Id: "type", Type: "string", Value: "response"}}
+		if len(r.Question) > 0 {
```

Comment:
Removed whole section and now we just change index 0
attrs[0].Value = "response"